src/topp/MascotAdapterOnline.cpp

```diff
@@ -200,8 +200,10 @@ class TOPPMascotAdapterOnline :
     }
   }
 
+  // merge b into a
   void mergeIDs_(ProteinIdentification& p_a, const ProteinIdentification& p_b, vector<PeptideIdentification>& pep_a, const vector<PeptideIdentification>& pep_b)
   {
+    // add proteins of b to a
     for (const ProteinHit& p : p_b.getHits())
```

Comment:
By the way, what about duplicate proteins? You do not want to have them since they will confuse a lot of algorithms!

Reply:
Good point. Currently, I think it is necessary to call PeptideIndexer anyway so maybe that is not a problem in practice.

Reply:
Ok yes, this should clear the proteins and re-add them uniquely based on the database in the Indexer.

Reply:
Actually, we could think about not adding any proteins at all then, to force the usage of PeptideIndexer.

Reply:
Yeah I think we could do that. Will probably change that when the other part is finished.